modules/aruco/tutorials/aruco_detection/aruco_detection.markdown

```diff
@@ -458,55 +467,61 @@ Detector Parameters
 One of the parameters of ```detectMarkers()``` function is a ```DetectorParameters``` object. This object
 includes all the options that can be customized during the marker detection process.
 
-In this section, all these parameters are commented. The parameters can be classified depending on
-the process they are involved:
+This section describes each detector parameter. The parameters can be classified depending on
+the process in which they’re involved:
 
-#### Thresholding
+### Thresholding
 
 One of the first steps of the marker detection process is an adaptive thresholding of the input image.
 
 For instance, the thresholded image for the sample image used above is:
 
 ![Thresholded image](images/singlemarkersthresh.png)
 
-This thresholding can be customized in the following parameters:
+This thresholding can be customized with the following parameters:
 
-- ```int adaptiveThreshWinSizeMin```, ```int adaptiveThreshWinSizeMax```, ```int adaptiveThreshWinSizeStep```
+- ```int adaptiveThreshWinSizeMin```
+- ```int adaptiveThreshWinSizeMax```
+- ```int adaptiveThreshWinSizeStep```
 
 The ```adaptiveThreshWinSizeMin``` and ```adaptiveThreshWinSizeMax``` parameters represent the interval where the
 thresholding window sizes (in pixels) are selected for the adaptive thresholding (see OpenCV
 ```threshold()``` function for more details).
```

Comment:
Please check [live demo](http://pullrequest.opencv.org/buildbot/export/pr_contrib/2357/docs/d5/dae/tutorial_aruco_detection.html). Page becomes broken here.

Perhaps is is due triple \`\`\` guards. Consider using single \` guard for items on the same line. Triple guards are used for multi-line code blocks.

Reply:
I thought those looked weird. They were in the original version. I replaced them with single back-ticks and that seems to have fixed it.